### DISCUSSION TITLE
W2

### DISCUSSION THREAD

**Discussion by JerryNixon** [ColoradoChristianUniversity/CIS-230-SP24#11] [General]
Hi!

**Comment by SeFreemanCCU** [comment]
string name = "Seth";
name.Display();

**Comment by M10377** [comment]
var name = "Micah";
name.Display()

**Comment by GabeWassmer** [comment]
string firstName = "Gabe";
firstName.Display();

string lastName = "Wassmer";
lastName.Display();

**Comment by masonlink17** [comment]
var firstName = "Mason";
firstName.Display();

var string lastName = "Link";
lastName.Display();

**Comment by mjs2002** [comment]
string first, middle, last;
first = "Marek";
middle = "James";
last = "Schroyer";
first + " " + middle + " " + last

**Comment by Alex-0-Baxter** [comment]
var first = "Alex";
var last = "Baxter";
first = first +" "+ last;
first.Display();

**Comment by syd-codes** [comment]
string first = "Sydney";
string last = " Crowder";

string fullName = string.Concat(first, last);
fullName.Display();

**Comment by 0578327** [comment]
String first = "Jaxson";
String last = " McAdoo";
String firstLast = String.Concat(first, last);
firstLast.Display();

**Comment by 0578327** [comment]
String first = "Jaxson";
String last = " McAdoo";
String firstLast = String.Format("{0} {1}", first, last);
firstLast.Display();

**Comment by jewelsmski** [comment]
string first = "Jewels"; 
        string last = "Marlewski"; 

        StringBuilder fullNameBuilder = new StringBuilder();
        fullNameBuilder.Append(first).Append(" ").Append(last);

        string Name = fullNameBuilder.ToString(); 
Name.Display();

**Comment by IDMunguia** [comment]
string first = "Diego";
string last = "Munguia";
StringBuilder fullname = new StringBuilder(first+" "+last);
fullname.Display();

**Comment by IDMunguia** [comment]
string first = "Diego";
string last = "Munguia";
string fullname = String.Join( " ", first, last);
fullname.Display();

**Comment by SeFreemanCCU** [comment]
string first, last;
first = "Seth";
last = "Freeman";
string name;
name += first;
name += " ";
name += last;
name.Display();

**Comment by M10377** [comment]
var first = "Micah";
var last = "Bissell";
var firstLast = $"{first} {last}";
firstLast.Display();

**Comment by 0578327** [comment]
for task 19

string name = "Jaxson McAdoo";
string nameLower = name.ToLower();
nameLower.Display();

**Comment by GabeWassmer** [comment]
var name = "Gabe Wassmer";
string text;
text = name.Substring(0, 4);
text.Display();

**Comment by jewelsmski** [comment]
for task 20

string name = "Jewels Marlewski";
string reversedName = new string(name.Reverse().ToArray());
reversedName.Display();

**Comment by Alex-0-Baxter** [comment]
// TASK 17
// using the string name= "Alex Baxter" use a REgular expression to assign the Word Alex to the variable "text"
 
using System.Text.RegularExpressions;
string name;
name = "Alex Baxter";
string text;
string pattern = @"^\w+";
Regex regex = new Regex(pattern);
text = regex.Match(name).Value;
Console.WriteLine(text);

**Comment by masonlink17** [comment]
var name = "Mason Link";
var nameParts = name.Split(' ');
var text = nameParts.Length > 0 ? nameParts[0] : " ";
Console.WriteLine(text);
